.travis.yml

```diff
@@ -22,9 +22,9 @@ before_install:
   - npm i -g jshint
 
 script:
+  - coala --non-interactive -V
   - ./.ci/build.sh
   - rm -rf private/
-  - coala --non-interactive -V
```

Comment:
Any reason for the reordering, or is it still WIP ?

Reply:
It is ready for review. It has passed Travis (it is failing due to API limit, but it has passed it a few times), but failed on Netlify (I am still investigating that).

The reordering is because of this issue https://github.com/coala/coala-bears/issues/2522. `python manage.py run_meta_review_system` will generate a large file (20MB), which makes `coala` gets stuck. The ordering eschews the problem. Also, I think there is justification to do so: we shouldn't run coala on files generated by `.ci/build.sh` IMO.

Reply:
I think it's worth adding a comment with a link to the issue, or just explain it in the commit.